store/sqlstore/status_store.go

```diff
@@ -37,6 +39,39 @@ func (s SqlStatusStore) createIndexesIfNotExists() {
 	s.CreateIndexIfNotExists("idx_status_status", "Status", "Status")
 }
 
+func statusSliceColumns() []string {
+	return []string{"UserId", "Status", "Manual", "LastActivityAt", "ActiveChannel", "DNDEndTimeUnix", "PrevStatus"}
+}
+
+func statusToSlice(status *model.Status) []interface{} {
+	return []interface{}{
+		status.UserId,
+		status.Status,
+		status.Manual,
+		status.LastActivityAt,
+		status.ActiveChannel,
+		status.DNDEndTimeUnix,
+		status.PrevStatus,
+	}
+}
+
+func (s SqlStatusStore) SaveMultiple(statuses []*model.Status) ([]*model.Status, error) {
+	builder := s.getQueryBuilder().Insert("Status").Columns(statusSliceColumns()...)
```

Comment:
I think this wrong. We should update, not insert. Anyways, we don't need this. See my comment above.